Remove fake tabs from install guides (#2390)
withastro/docs: src/pages/en/install/manual.mdx

layout: ~/layouts/MainLayout.astro
i18nReady: true
---
import Button from '~/components/Button.astro'
import FileTree from '~/components/FileTree.astro';
import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'

> Ready to install Astro? Follow this manual set-up guide to get started.

Reviewer: ```suggestion
This guide will walk you through the steps to manually install and configure a new Astro project.
```
I'm thinking something less "encouraging" here, to nudge people towards the wizard unless they're darn sure they want to go through the manual process.

Author: Would this be overkill? I’m cool with your version if so.

```suggestion
This guide will walk you through the steps to manually install and configure a new Astro project if you prefer not to use [the automatic CLI tool](/en/install/auto/).
```

> #### Want a quick way to get started?

Reviewer: ```suggestion
#### Want to let Houston do the work for you instead?
```
Too cheesy?  :sweat_smile:

Reviewer: > Want to let our automated assistant Houston do the work for you instead?

As I wrote that, I was like, "ugh, I hate automated assistants" so NOT that.

But maybe something like: Want a guided CLI experience?

Author: If we want comparative maybe just?
```suggestion
#### Want a quicker way to get started?
```

Author: I don’t think the CLI part is really the important thing, it’s more the speed/convenience angle that’s attractive. Plus “CLI wizard” is in the button text. I thought “quick and easy” initially but usually avoid “easy” if I can.

Reviewer: Yeah, my objection was that I'm not sure this clearly enough says that it's a REPLACEMENT for doing the steps below. For someone who doesn't know any better, this could be a way to start THE STEPS BELOW.

Reviewer: > Prefer a guided installation experience?

Author: Ahhh, missed that nuance. How about…

```suggestion
#### Want a quicker way to get started instead?
```

Author: Ooh, I do like “prefer” to help communicate this.

Author: Maybe?

```suggestion
#### Prefer a quicker way to get started?
```
